JWBestPracticeApps/Podfile

```diff
@@ -50,6 +50,8 @@ end
 target 'DRM Fairplay' do
   project 'DRM Fairplay/DRM Fairplay'
   common_JWPlayer
+  pod 'SwiftJWT'
```

Comment:
I think we should keep pod dependency to the minimal which is strictly required by the SDK for ads, casting etc. We could do this for the Advance Implementation we were discussing previously.